### ISSUE TITLE
Accept byte array directly instead of a file path

### ISSUE THREAD

**Issue by ericdunn** [GowenGit/docnet#2]
It would be nice if we could pass in a pdf's byte[] directly to GetDocReader, instead of a file path. In my case, all I have is the byte array, so I need to first create a temp file with this, then pass that in, then ensure I delete it (I used the same idea as your test case with the TempFile).

I don't have time to actually create a full pull request, but I've tested the following addition to DocReader.cs and it seemd to work just fine: 

```
 public DocReader(byte[] fileData, string password, int dimOne, int dimTwo)
        {
            _dimOne = dimOne;
            _dimTwo = dimTwo;
            lock (DocLib.Lock)
            {
                IntPtr unmanagedPointer = Marshal.AllocHGlobal(fileData.Length);
                Marshal.Copy(fileData, 0, unmanagedPointer, fileData.Length);
                var d = fpdf_view.FPDF_LoadMemDocument(unmanagedPointer, fileData.Length, password);
                Marshal.FreeHGlobal(unmanagedPointer);
                _docWrapper = new DocumentWrapper(d);
            }
        }

```
Obviously you probably want to add more error checking etc, but that does work fine.

Otherwise, your library solves a huge gap in dotnet core right now around dealing with PDF files!

Kudos!
Eric.

**Comment by Modest-as** [comment]
Hey Eric,

I think that this would be a great addition to the API and was thinking of adding it myself if people actually use it. I will try and implement this at some point this week and release a new nuget package version.

Thanks,

Modestas

**Comment by ericdunn** [comment]
Excellent thanks! I'll keep an eye out for it!

**Comment by Modest-as** [comment]
Just released a new nuget version `1.1.0` with all necessary changes.

**Comment by ericdunn** [comment]
Excellent thanks! I'll update tomorrow and let you know how it goes !
